pkg/types/installconfig.go

```diff
@@ -15,14 +15,16 @@ import (
 	"github.com/openshift/installer/pkg/types/openstack"
 	"github.com/openshift/installer/pkg/types/ovirt"
 	"github.com/openshift/installer/pkg/types/vsphere"
+	"github.com/pkg/errors"
```

Comment:
This should go in a new section between the golang imports and the installer imports. But I don't think you need this import anyway as you can use `fmt.Errorf` instead of `errors.Errorf`.